src/reference/modules/reference/pages/variables.adoc

```diff
@@ -541,6 +541,19 @@ site-specific elements...
 
 NOTE: These variables are not available on Windows nodes, but only on with the classic Linux/AIX agent and within the `Technique Editor`.
 
+==== Technique parameters
```

Comment:
Could you also add that this prefixed syntax is required when the parameter is used outside of the technique: in a template or for generic methods taking a variable name (and not a value)

Reply:
sure